src/generators/ts/ts.generator.ts

```diff
@@ -1,4 +1,4 @@
-import { parse } from 'path';
+import { parse, sep, join, posix } from 'path';
```

Comment:
I think `sep` is unused now that you switched to using `posix.sep`, so can be removed from the imports.